.github/workflows/scripts/nugetSlackNotifications/packageInfo.json

```diff
@@ -85,10 +106,13 @@
     {
         "packageName": "nest"
     },
+    {
```

Comment:
Is this so we update the API reference in our tests? Should we make that part of the Release process?

Reply:
Yeah, that's the intention. 

I don't know that it needs to be part of the release process - unless we change the API (which is almost never), it doesn't matter if the package gets updated right away or not.